exporter.go

```diff
@@ -188,9 +189,7 @@ func main() {
 
 	go func() {
 		log.Infoln("Starting server on", *listenAddress)
-		if err := http.ListenAndServe(*listenAddress, nil); err != nil {
-			log.Fatalf("cannot start WMI exporter: %s", err)
-		}
+		log.Fatalf("cannot start WMI exporter: %s", http.ListenAndServe(*listenAddress, nil))
```

Comment:
Hi @martinlindhe 
`http.ListenAndServer` only returns when an *error* occured. I.e. it never returns *nil*. This style is also used in the official [Go http documentation](https://golang.org/pkg/net/http/)